apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -412,46 +401,11 @@ class CensusForm extends Component {
               </label>
             </div>
           )}
-          {!this.state.showSchoolYearDropdown && (
-            <div>
-              <div style={styles.question}>
-                Please answer the questions below about the{' '}
-                {this.props.initialSchoolYear}-
-                {this.props.initialSchoolYear + 1} school year. (
-                <a onClick={this.showSchoolYearDropdown}>
-                  Answer for a different school year.
-                </a>
-                )
-              </div>
-              <input
-                type="hidden"
-                id="school_year"
-                name="school_year"
-                value={this.props.initialSchoolYear}
-              />
```

Comment:
The presence of this hidden <input> field is what previously set the current census year for the census form (if the user did not open the year dropdown)